### PR TITLE
Contribute with new post

### PR DESCRIPTION
**Pull request by leandrosimoes**
## TL;DR;

Three very useful and syntax sugar hacks to speed up your development.
## Username

[@leandrosimoes](https://github.com/leandrosimoes)

### CODE REVIEW

File: _posts/en/2016-xx-xx-three-useful-hacks.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+
+title: Three useful hacks
+tip-number: 00
+tip-username: leandrosimoes 
+tip-username-profile: https://github.com/leandrosimoes
+tip-tldr: Three very useful and sintax sugar hacks to speed up your development.
```
Comment (janosorcsik) [inline _posts/en/2016-xx-xx-three-useful-hacks.md]: Typo: sintax → syntax

**Review by janosorcsik — CHANGES_REQUESTED** [review]

File: _posts/en/2016-xx-xx-three-useful-hacks.md (hunk 1 above)
Comment (leandrosimoes) [reply _posts/en/2016-xx-xx-three-useful-hacks.md]: Done! Tks!

File: _posts/en/2016-xx-xx-three-useful-hacks.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+tip-username: leandrosimoes 
+tip-username-profile: https://github.com/leandrosimoes
+tip-tldr: Three very useful and syntax sugar hacks to speed up your development.
+
+
+categories:
+    - en
+---
+
+#### Convert to boolean using "!!"
```
Comment (loverajoel) [inline _posts/en/2016-xx-xx-three-useful-hacks.md]: this already exists http://www.jstips.co/en/converting-truthy-falsy-values-to-boolean/

Comment (leandrosimoes) [reply _posts/en/2016-xx-xx-three-useful-hacks.md]: Tks man! I changed it to another tip.

**Pull request merged**